template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/IdentityServiceConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Company.WebApplication1.Identity
+{
+    public class IdentityServiceConstants
```

Comment:
Are these all actual constants in the sense that they are immutable? They will never change?